R/utils.R

```diff
@@ -650,7 +650,7 @@ remove_dup_cols <- function(data){
     dup_cols <- names(data)[duplicated(names(data))]
     dup_cols_str <- paste(dup_cols, collapse = ", ")
     dup_id <- c(which(duplicated(names(data))) -1, which(duplicated(names(data)))) %>% sort()
-    warning(glue("{basename(.path)} had the following duplicated columns: {dup_cols_str}\n  Duplicate names will be repaired with `make.unique()`")) # something like this, and then say if they're equal
+    warning(glue("Following duplicated columns: {dup_cols_str}\n  Duplicate names will be repaired with `make.unique()`")) # something like this, and then say if they're equal
     colnames(data) <- make.unique(colnames(data) )
```

Comment:
@rymarinelli can you remove this comment before merging?

Reply:
In case its not clear, just this comment: `# something like this, and then say if they're equal`

Reply:
I added an extra comment at the end. Is it what you had in mind?

Reply:
I just meant remove the previous one (see above), but the comment you added is fine